Update audit-logs.adoc (#8405)
circleci/circleci-docs: jekyll/_cci2/audit-logs.adoc

[#audit-log-events]
== Audit log events

> The following are a list of common and important events found in the audit log. This list is not comprehensive and you may see additional action types logged that are not represented below. See `action` in the Field section below for the definition and format.

Reviewer: ```suggestion
The following list shows common and important events found in the audit log. This list is not comprehensive, and you may see additional action types logged that are not represented below. See `action` in the Field section below for the definition and format.
```
